feat: fluttercon usa 2025 sponsorship blog post (#291)
shorebirdtech/website: src/content/blog/flutterconusa-2025-sponsor.md

---
title: Shorebird Sponsors Fluttercon USA 2025
author: shorebirdtech
description:
  Join us June 25–26 to connect with the Flutter community in North America.
date: 2025-05-06
cover: flutterconusa-2025-cover.png
---

It’s been a year full of in person conferences and meetups for us and we are
just getting started. It’s amazing to be able to get out into the community and
see how Shorebird is being used but also hearing feedback from current and
prospective customers. It seems like once developers realize the tools that

> Shorebird adds into your Flutter toolbox they can’t help but want to get it

Reviewer: ```suggestion
Shorebird adds to their Flutter toolbox they can’t help but want to get it
```
